Python/compile.c

```diff
@@ -7869,42 +7869,6 @@ makecode(struct compiler *c, struct assembler *a, PyObject *constslist,
     return co;
 }
 
-
-/* For debugging purposes only */
-#if 0
-static void
-dump_instr(struct instr *i)
```

Comment:
These have been useful occasionally, so I would rather not remove them.

Reply:
Restored.